File: ProjectWebView/app/src/main/java/com/aplikasi/projectwebview/MainActivity.java
package com.aplikasi.projectwebview;

import androidx.appcompat.app.AppCompatActivity;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {
    WebView web;
    WebSettings websettings1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        web = (WebView)findViewById(R.id.testWebView);
        websettings1 = web.getSettings();

        web.setWebViewClient(new WebViewClient());
        web.loadUrl("https://sites.google.com/view/jualskincaretermurah2020/halaman-muka");
    }
}

File: Pertemuan8/app/src/main/java/com/aplikasi/pertemuan8/MainActivity.kt
package com.aplikasi.pertemuan8

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    private fun myToast(pesan: String, waktu:Int)
            = Toast.makeText(this,pesan,waktu).show()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        

    }
    fun showToast(view: View) {
            myToast("Haloo",Toast.LENGTH_SHORT)
    }
}